test/e2e/artifact_test.go

```diff
@@ -257,6 +245,12 @@ var _ = Describe("Podman artifact", func() {
 		// There should be no artifacts in the store
 		rmAll := podmanTest.PodmanExitCleanly("artifact", "ls", "--noheading")
 		Expect(rmAll.OutputToString()).To(BeEmpty())
+
+		// Trying to remove an artifact that does not exist should pass with -i
+		podmanTest.PodmanExitCleanly("artifact", "rm", "-i", "foobar")
+
+		// Trying to remove an existing artifact should also pass with -i
+		podmanTest.PodmanExitCleanly("artifact", "rm", "-i", artifact1Name)
```

Comment:
here it would be nice to actually check that the artifact has been removed so that the option is not a NOP.